Lib/test/test_platform.py

```diff
@@ -219,6 +219,19 @@ def test_uname(self):
         self.assertEqual(res[-1], res.processor)
         self.assertEqual(len(res), 6)
 
+        if os.name == "posix":
+            uname = os.uname()
+            self.assertEqual(res.node, uname.nodename)
+            self.assertEqual(res.version, uname.version)
+            self.assertEqual(res.machine, uname.machine)
+
+            if sys.platform == "android":
+                self.assertEqual(res.system, "Android")
+                self.assertEqual(res.release, platform.android_ver().release)
+            else:
+                self.assertEqual(res.system, uname.sysname)
+                self.assertEqual(res.release, uname.release)
+
         if sys.platform == "ios":
             self.assertIn(res.system, {"iOS", "iPadOS"})
             self.assertEqual(res.release, platform.ios_ver().release)
```

Comment:
`os.name == "posix"` on iOS as well, so this should be merged into the previous block.

Reply:
Ah, the perils of in-browser merging. Corrected in latest commit.